lib/Stampie/Mailer.php

```diff
@@ -50,6 +60,26 @@ public function getAdapter()
     }
 
     /**
+     * @return MessageFactory
+     */
+    private function getMessageFactory()
+    {
+        return $this->messageFactory ?: MessageFactoryDiscovery::find();
```

Comment:
Also these are missing from the Interface. However i am not fund of changing the interface too much.